examples/data/catagory-data-view-all-modes.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
```

Comment:
Let's move dataviews to a new folder in examples, named `dataview` as the namespace, instead of data. Also there is a typo in the file name --> 'catagory'

Don't worry about viewport-features, I have updated it and moved it to another folder in my WIP pull on examples